objectivec/GPBDescriptor_PackagePrivate.h

```diff
@@ -69,6 +69,10 @@ typedef NS_OPTIONS(uint16_t, GPBFieldFlags) {
   GPBFieldMapKeySFixed64 = 10 << 8,
   GPBFieldMapKeyBool     = 11 << 8,
   GPBFieldMapKeyString   = 12 << 8,
+
+  // This is used as a stopgap as we move from using class names in the
+  // GPBMessageFieldDescriptions to using class references.
+  GPBFieldUsesClazz       = 1 << 12
```

Comment:
Did you look at using GPBDescriptorInitializationFlags so it just gets passed to the MessageDescriptor?  You'd still have to relay it to the fields, but it keeps things out of the field flags for the time being.  Not sure if that's better or not, just figured I'd call it out incase you didn't look at it.

Reply:
I did.. I felt that temporarily hijacking the field flags was clearer and easier to test. I needed to do it for extensions as well in an upcoming CL.

Reply:
While I'd love this to really be _temporarily_, the truth is it will live until we decide to take a breaking change.  🙁

Reply:
Moved things over to GPBDescriptorInitializationFlags